backend/SlothOrganizer/SlothOrganizer.Services.Tests.Unit/Auth/AuthServiceTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿using FakeItEasy;
+using SlothOrganizer.Contracts.DTO.Auth;
+using SlothOrganizer.Contracts.DTO.User;
+using SlothOrganizer.Domain.Exceptions;
+using SlothOrganizer.Services.Abstractions.Auth.Tokens;
+using SlothOrganizer.Services.Abstractions.Auth.UserVerification;
+using SlothOrganizer.Services.Abstractions.Users;
+using SlothOrganizer.Services.Auth;
+using Xunit;
+
+namespace SlothOrganizer.Services.Tests.Unit.Auth
+{
+    public class AuthServiceTests
+    {
+        private readonly AuthService _authService;
+        private readonly IUserService _userService;
+        private readonly IAccessTokenService _accessTokenService;
+        private readonly IRefreshTokenService _refreshTokenService;
+        private readonly INotificationService _notificationService;
+
+        public AuthServiceTests()
+        {
+            _userService = A.Fake<IUserService>();
+            _accessTokenService = A.Fake<IAccessTokenService>();
+            _refreshTokenService = A.Fake<IRefreshTokenService>();
+            _notificationService = A.Fake<INotificationService>();
+
+            _authService = new AuthService(_accessTokenService,
+                _userService,
+                _notificationService,
+                _refreshTokenService);
+        }
+
+        [Fact]
+        public async Task VerifyEmail_WhenValid_ShouldVerify()
```

Comment:
As far as I understood, the main logic for email verification is implemented on the database level. So, this unit test tests the stub configuration?

Reply:
It would be better to move the such a test to the integration level.

Reply:
There already is a corresponding integration test for email verification. I agree that this particular method doesn't have much logic to test (it just throws an exception in case of failure or returns data if everything is ok), but I think it wouldn't hurt to have all public methods covered with unit tests as well